Description: Do not print standard multiarch path
 {/usr,}/lib/i386-linux-gnu are in the standard search path, there is no
 need to explicitely point gcc there with a -L argument.
 Also we are installing the actual library and the so-symlink in different
 locations which makes this incorrect no matter which one we choose.
Origin: vendor
Forwarded: not-needed

--- libgcrypt11-1.5.2.orig/src/libgcrypt-config.in
+++ libgcrypt11-1.5.2/src/libgcrypt-config.in
@@ -154,7 +154,7 @@ if test "$echo_libs" = "yes"; then
     libs_final="$libs"
 
     # Set up `libdirs'.
-    if test "x$libdir" != "x/usr/lib" -a "x$libdir" != "x/lib"; then
+    if test "x$libdir" != "x/usr/lib" -a "x$libdir" != "x/lib" -a "x$libdir" != "x/lib/`dpkg-architecture -qDEB_HOST_MULTIARCH`" ; then
 	libdirs="-L$libdir"
     fi
 
